internal/webhook/apps/v1alpha1/nimservice_webhook_validation_helper.go

```diff
@@ -69,10 +69,17 @@ func validateNIMServiceSpec(spec *appsv1alpha1.NIMServiceSpec, fldPath *field.Pa
 	errList = append(errList, validateExposeConfiguration(&spec.Expose, fldPath.Child("expose").Child("ingress"))...)
 	errList = append(errList, validateMetricsConfiguration(&spec.Metrics, fldPath.Child("metrics"))...)
 	errList = append(errList, validateScaleConfiguration(&spec.Scale, fldPath.Child("scale"))...)
-	errList = append(errList, validateResourcesConfiguration(spec.Resources, fldPath.Child("resources"))...)
+	errList = append(errList, validateResourcesConfiguration(spec, fldPath.Child("resources"))...)
```

Comment:
```suggestion
	errList = append(errList, validateResourcesConfiguration(spec, fldPath)...)
```
nit: its better to match the fieldPath with the input in the caller.

Reply:
Fixed